### ISSUE TITLE
Piwik v3.5 broke

### ISSUE THREAD

**Issue by roysoham** [matomo-org/matomo#12914]
A fatal error occurred
The following error just broke Piwik (v3.5.0):

Class 'Piwik\Plugins\PrivacyManager\Model\LogDataAnonymizations' not found
in

/usr/share/nginx/analytics/core/Updates/3.5.0-b2.php line 48
Troubleshooting
Follow these steps to solve the issue or report it to the team:
If you have just updated Piwik to the latest version, please try to restart your web server. This will clear the PHP opcache which may solve the problem.
If this is the first time you see this error, please try refresh the page.
If this error continues to happen, we appreciate if you send the error report to the Piwik team.

**Comment by fdellwing** [comment]
Did you restart the web server? Also show output of `ls -la /usr/share/nginx/analytics/plugins/PrivacyManager/Model`.

**Comment by roysoham** [comment]
Yes two files are there : LogDataAnonymizations.php, DataSubjects.php

**Comment by sgiehl** [comment]
maybe try reuploading all matomo files to ensure all files are correct...

**Comment by mattab** [comment]
@roysoham what is the output of `ls -la /usr/share/nginx/analytics/plugins/PrivacyManager/Model` ?
